homeassistant/components/homewizard/sensor.py

```diff
@@ -119,6 +119,20 @@
         device_class=SensorDeviceClass.GAS,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    SensorEntityDescription(
+        key="active_liter_lpm",
+        name="Active Water Usage",
+        native_unit_of_measurement="l/m",
```

Comment:
Can/do we need to add this unit of measurement to [homeassistant/const.py](https://github.com/home-assistant/core/blob/dev/homeassistant/const.py)?

Reply:
That should not be needed. We move it to const.py when it is more broadly used or when we start doing conversions.

Nevertheless, I do think the unit is wrong, as it is not liter per meter 😉 

```suggestion
        native_unit_of_measurement="l/min",
```

Reply:
Ha, that is one way so see it 👍 

I took that notation from const.py (`VOLUME_FLOW_RATE_CUBIC_FEET_PER_MINUTE: Final = "ft³/m"`). But l/min is better.